lsp-mode.el

```diff
@@ -3240,7 +3240,7 @@ Return a nested alist keyed by symbol names. e.g.
 
 (defun lsp-server-present? (final-command)
   "Check whether FINAL-COMMAND is present."
-  (executable-find (nth 0 final-command)))
+  (executable-find (nth 0 final-command) (file-remote-p default-directory)))
```

Comment:
Here I get:

```
(wrong-number-of-arguments (1 . 1) 2)
  executable-find("/home/mj/cquery/build/release/bin/cquery" "/ssh:someserver.se:")
  lsp-server-present\?(("/home/mj/cquery/build/release/bin/cquery"))
```
Maybe missing something obvious?

#### System Info :computer:
- OS: gnu/linux
- Emacs: 26.1
- Spacemacs: 0.300.0
- Spacemacs branch: 20181231 (rev. 2c28c7d21)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
(helm auto-completion emacs-lisp git markdown multiple-cursors org
      (shell :variables shell-default-height 30 shell-default-position 'bottom)
      spell-checking syntax-checking
      (version-control :variables version-control-diff-tool 'diff-hl version-control-global-margin t version-control-diff-side 'left)
      (c-c++ :variables c-c++-backend 'lsp-cquery c-c++-lsp-executable "/home/mj/cquery/build/release/bin/cquery" c-c++-default-mode-for-headers 'c++-mode)
      themes-megapack go
      (plantuml :variables plantuml-jar-path "/usr/share/plantuml/plantuml.jar")
      yaml)
```
- System configuration features: XPM JPEG TIFF GIF PNG RSVG IMAGEMAGICK SOUND GPM DBUS GSETTINGS NOTIFY LIBSELINUX GNUTLS LIBXML2 FREETYPE M17N_FLT LIBOTF XFT ZLIB TOOLKIT_SCROLL_BARS GTK3 X11 MODULES THREADS XWIDGETS LIBSYSTEMD LCMS2

Reply:
This is indeed a post-26.1 feature. Added a fallback implementation for current releases.